invenio/modules/workflows/models.py

```diff
@@ -682,16 +727,15 @@ def change_status(self, message):
 
     def get_current_task(self):
         """Return the current task from the workflow engine for this object."""
-        extra_data = self.get_extra_data()
         try:
-            return extra_data["_task_counter"]
+            return self.extra_data["_task_counter"]
         except KeyError:
             # Assume old version "task_counter"
-            return extra_data["task_counter"]
+            return self.extra_data["task_counter"]
```

Comment:
Shall we define `task_counter` and `action` as direct column types in the `DbWorkflowObject` model? They seem quite important. `action` is important for Holding Pen, `task_counter` is important for the workflow itself.